vars/uploadMinSnapshotsToS3.groovy

```diff
@@ -57,18 +55,23 @@ void call(Map args = [:]) {
             String corePluginDir = "${WORKSPACE}/${distribution}/builds/${productName}/core-plugins".replace("\\", "/")
             boolean corePluginDirExists = fileExists(corePluginDir)
             if (architecture == "x64" && platform == "linux" && distribution == "tar" && corePluginDirExists) {
-                echo("Create .sha512 for Core Plugins Snapshots")
-                argsMap['artifactPath'] = corePluginDir
+                echo("Create .sha512 and .sig for Core Plugins Snapshots")
+                fileActions = [createSha512Checksums(), createSignatureFiles()]
+                argsMapPlugins = [:]
+                argsMapPlugins['sigtype'] = '.sig'
```

Comment:
Why are we signing the artifacts? I believe we are only create checksums  for these artifacts.

Reply:
Because we have to have sigs to install them as core checks it:
https://github.com/opensearch-project/opensearch-build/issues/5096#issuecomment-2459880876